bHY6yh-de1103/cod-yyyiou_bHY6yh-de1103.xml

```diff
@@ -0,0 +1,152 @@
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 5</title>
+               <date when="2023-06-02">June 2, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-02" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#V" n="2-v"/>
+            <cb ed="#V" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de1103"><!-- p1l1-r5 -->
+            <head xml:id="bHY6yh-de1103-Hd1e108">Rubrica 5</head>
+            <head xml:id="bHY6yh-de1103-Hd1e111" type="question-title">De retributione Dei</head>
+            <p xml:id="bHY6yh-de1103-d1e114">
+               De retributione dei Rubrica. 5.
+            </p>
+            <p xml:id="bHY6yh-de1103-d1e119">
+              <ref xml:id="bHY6yh-de1103-Rd1e116">Iob ystoria 
+              <lb ed="#C"/>XXXIIIIo</ref> <quote xml:id="bHY6yh-de1103-Qd1e121"> 
+              <lb ed="#C"/>Opus hominis reddit ei dominus et
+              <lb ed="#C"/> iusta uris singulorum restituet</quote>
+            </p>
+            <p xml:id="bHY6yh-de1103-d1e131">
+              ¶ <ref xml:id="bHY6yh-de1103-Rd1e132">In
+              <lb ed="#C"/>genesi IIIIo capitulo </ref> <quote xml:id="bHY6yh-de1103-Qd1e135">nonne si bene egeris<sic>recapies</sic></quote>
+            </p>
+            <p xml:id="bHY6yh-de1103-d1e142">
+              ¶ <ref xml:id="bHY6yh-de1103-Rd1e145">Psalmista LXI</ref> <quote xml:id="bHY6yh-de1103-Qd1e146"> Dominis reddet <sic>vnianque</sic> iuxta 
+              <lb ed="#C"/>opera sua. </quote>
+            </p>
+            <p xml:id="bHY6yh-de1103-d1e152"> /
```

Comment:
These slashes throughout look like left over punctuation from the Venice edition. Can you double check. If they are supposed to be there can you just confirm? If they are not supposed to be there, please delete.

```suggestion
            <p xml:id="bHY6yh-de1103-d1e152"> /
```